nssp/params.json.template

```diff
@@ -0,0 +1,13 @@
+{
+  "common": {
```

Comment:
issue: We will also need a template params in [the `ansible/templates` directory](https://github.com/cmu-delphi/covidcast-indicators/tree/414638c9c284c76d5416ad8388ae6fb30e69583a/ansible/templates).

Reply:
yep. That one is [here](https://github.com/cmu-delphi/covidcast-indicators/pull/1950)